lib/activeadmin_settings_cached.rb

```diff
@@ -1,4 +1,11 @@
-require "activeadmin_settings_cached/engine"
```

Comment:
Hm. `rails plugin new activeadmin_settings_cached --mountable` generated with this `require`